modules/statics/src/account.ts

```diff
@@ -558,15 +553,16 @@ export function tronToken(
   features: CoinFeature[] = AccountCoin.DEFAULT_FEATURES,
   prefix: string = '',
   suffix: string = name.toUpperCase(),
-  network: AccountNetwork = Networks.main.trx,
-  primaryKeyCurve: KeyCurve = KeyCurve.Secp256k1,
-  feeLimit: string
+  network: TronNetwork = Networks.main.trx,
+  primaryKeyCurve: KeyCurve = KeyCurve.Secp256k1
 ) {
+  const tokenNetwork = { ...network };
```

Comment:
We can't do this - we rely on the coin network being the same object instance as those exported by the library

Reply:
We have added another field to the network object for the contract call default fee limit. Both values are required to be on Statics.